Compiling 1.7.0 on Linux without USB support (./configure --without-libusb) causes compiler error in jeeps/gpsusbstub.cc

Using RedHat Enterprise Linux 7.

Downloaded the latest source version 1.7.0 from http://www.gpsbabel.org/download.html.  

Build page recommends to "**yum install qt-develop**" but when attempted "**./configure**" got an error:

`configure: error: Qt qmake (with QT_VERSION >= 5.9) is required but was not found.`

Found that using "**yum install qt5-qtbase-devel**" resolved the QT version issue.

Running "**make**" generated an error:

`g++  -g -O2 -Wall -fPIC -c   -I. -I./zlib -I/usr/include/qt5  -DHAVE_CONFIG_H jeeps/gpsusbstub.cc -o jeeps/gpsusbstub.o
jeeps/gpsusbstub.cc: In function ‘int gusb_init(const char*, gpsdevh**)’:
jeeps/gpsusbstub.cc:37:9: error: ‘Fatal’ was not declared in this scope
   Fatal() << no_usb;
         ^
Makefile:132: recipe for target 'jeeps/gpsusbstub.o' failed
make: *** [jeeps/gpsusbstub.o] Error 1
`

Appears that a change was made to other source files that was missed in jeeps/gpsusbstub.cc.

1.7.0 jeeps/gpsusbstub.cc code is:

`Fatal() << no_usb;`

Resolved by changing to:

`fatal(FatalMsg() << no_usb);`

A different solution already appears to have been made to the current github source. 

You may want to include this as documentation for 1.7.0.

--- a/xmldoc/chapters/build.xml
+++ b/xmldoc/chapters/build.xml
@@ -35,7 +35,7 @@ You can grab a release from the <ulink url="https://www.gpsbabel.org/download.ht
 <para>
 Qt version 5.12 or newer is required for all builds.  MacOS and Windows users can download
 binaries from <ulink url="http://qt-project.org/downloads">Qt Downloads</ulink>
-Fedora or CentOS users may need to 'yum install qt-devel'.  Optionally, use qt5-base.  When in doubt, 'yum search qt' or 'yum search qt5' may help you find the correct package name. Ubuntu users may need to 'apt-get install qtbase5-dev'. Package names
+Fedora or CentOS users may need to 'dnf install qt5-qtbase-devel'.  When in doubt, 'dnf search qt' or 'dnf search qt5' may help you find the correct package name. Ubuntu users may need to 'apt-get install qt5-default'. Package names
 and versions in Linux frequently change, so you may need to ask your Linux vendor
 for help or look in tools/Docker* for inspiration for our automated builds that
 use Docker.

